constants.js

```diff
@@ -0,0 +1,23 @@
+const SERVER_EVENTS = {
```

Comment:
I'd prefer we don't make global constants file, but scope constants to the relevant modules. So that e.g. `TWO_FACTOR_AUTH_KINDS` live somewhere close to 2fa implementation.